extensions/amp-analytics/amp-analytics.md

```diff
@@ -14,11 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# <a name="amp-analytics"></a>`amp-analytics`
+### <a name="amp-analytics"></a>`amp-analytics`
```

Comment:
I changed it to be consistent with the rest of the extension .md files. They all currently start at level 3 and the main sections are level 4. I could see changing this back in the future, but for now wanted to get it consistent with the rest of the files.